lib/AST/ExternalASTMerger.cpp

```diff
@@ -57,7 +57,14 @@ LookupSameContext(Source<TranslationUnitDecl *> SourceTU, const DeclContext *DC,
   }
   auto *ND = cast<NamedDecl>(DC);
   DeclarationName Name = ND->getDeclName();
-  Source<DeclarationName> SourceName = ReverseImporter.Import(Name);
+  auto NameOrErr = ReverseImporter.Import(Name);
+  if (auto Err = NameOrErr.takeError()) {
+    // Ignore error.
+    handleAllErrors(std::move(Err), [](const ImportError&) { });
```

Comment:
Ideally we would do `return Err`, but that would require changing the signature of `LookupSameContext()` too.
I think, it is perfectly fine to change the signature of the `Import` and `Visit` functions first, then later we could address these kind of auxiliary functions too.